storagemarket/impl/clientstates/client_fsm.go

```diff
@@ -104,15 +105,22 @@ var ClientEvents = fsm.Events{
 		}),
 
 	fsm.Event(storagemarket.ClientEventDataTransferInitiated).
-		FromMany(storagemarket.StorageDealStartDataTransfer).To(storagemarket.StorageDealTransferring).
+		FromMany(storagemarket.StorageDealTransferQueued).To(storagemarket.StorageDealTransferring).
 		Action(func(deal *storagemarket.ClientDeal, channelId datatransfer.ChannelID) error {
 			deal.TransferChannelID = &channelId
 			deal.AddLog("data transfer initiated on channel id <%s>", channelId)
 			return nil
 		}),
 
+	fsm.Event(storagemarket.ClientEventDataTransferQueued).
+		FromMany(storagemarket.StorageDealStartDataTransfer).To(storagemarket.StorageDealTransferQueued).
+		Action(func(deal *storagemarket.ClientDeal, channelId datatransfer.ChannelID) error {
+			deal.AddLog("data transfer queued on channel id <%s>", channelId)
+			return nil
+		}),
+
```

Comment:
nit: can we move this block above the block that starts with
`fsm.Event(storagemarket.ClientEventDataTransferInitiated)`
just so that the events in the FSM are in the order that they will typically occur.